crates/sui-framework/packages/sui-system/sources/staking_pool.move

```diff
@@ -77,7 +77,7 @@ module sui_system::staking_pool {
     }
 
     /// A self-custodial object holding the staked SUI tokens.
-    struct StakedSui has key {
+    struct StakedSui has key, store {
         id: UID,
         /// ID of the staking pool we are staking with.
         pool_id: ID,
```

Comment:
Do we have getters for StakedSui contents?

Reply:
Yes we have getters for StakedSui